components/SightingList.jsx

```diff
@@ -56,43 +56,46 @@ export default SightingList = ({ navigation }) => {
         <View style={styles.titleContainer}>
           <Text style={styles.titleText}>All Sightings</Text>
         </View>
-        {loading && ( <Text style={styles.loadingText}>Loading...Please Wait</Text> )}
+        {loading && (
+          <Text style={styles.loadingText}>Loading...Please Wait</Text>
+        )}
 
         <View style={styles.buttonContainer}>
-          <TouchableOpacity style={styles.button} onPress={() => navigation.goBack()}>
+          <TouchableOpacity
+            style={styles.button}
+            onPress={() => navigation.goBack()}
+          >
             <Text style={styles.buttonText}>Go Back</Text>
           </TouchableOpacity>
         </View>
 
         <View style={styles.listContainer}>
           {allSightings.map((bird, index) => (
-              <TouchableOpacity key={index}
-                style={styles.birdCardContainer}
-                onPress={() => {navigation.navigate("Sighting", bird);}}
-              >
-                <View style={styles.birdCardImageContainer}>
-                  {bird.sighting_img_url === "" ? (
-                    <Image
-                      source={require("../assets/default-sighting-img.jpg")}
-                      style={[styles.birdCardImage]}
-                    />
-                  ) : (
-                    <Image
-                      source={{ uri: bird.sighting_img_url}}
-                      style={styles.birdCardImage}
-                    />
-                  )}
-                  
-                </View>
+            <TouchableOpacity
+              key={index}
+              style={styles.birdCardContainer}
+              onPress={() => {
+                navigation.navigate("Sighting", bird);
+              }}
```

Comment:
I think this has caused a conflict. Apart from that all looks fine. Where you have 'allSightings' on line 73 this should be 'birdsByDistance' however the line has changed.

Reply:
@kzzztkt I pulled this from main before the change was made so need to resolve the conflicts in the next step before merging